src/main/java/org/kohsuke/github/GHTeam.java

```diff
@@ -130,6 +130,19 @@ public void setPrivacy(Privacy privacy) throws IOException {
         root.createRequest().method("PATCH").with("privacy", privacy).withUrlPath(api("")).send();
     }
 
+    /**
+     * Retrieves the discussions.
+     *
+     * @return the paged iterable
+     * @throws IOException
+     *             the io exception
+     */
+    public PagedIterable<GHDiscussion> listDiscussions() throws IOException {
+        return root.createRequest()
+                .withUrlPath(api("/discussions"))
+                .toIterable(GHDiscussion[].class, item -> item.wrapUp(root));
```

Comment:
```suggestion
        return root.createRequest()
                .withUrlPath(api("/discussions"))
                .toIterable(GHDiscussion[].class, item -> item.wrapUp(this));
```

Also, I'd like the request to be in `GHDiscussion` as `readAll(GHTeam)`, like in [the `GHLabel` changes](https://github.com/hub4j/github-api/pull/724/files#r386305383).  I know this is an extra method call, but `GHDiscussion` should be the class where this work is really done. 

If you want to refactor `GHTeam.api()` with a better name and make it internal instead of `private` that's fine.